### PR TITLE
Create Harvard FAS Research Computing.yaml

### PR DESCRIPTION
**Pull request by dancaunt**
Add Frontier Squid resource group for Harvard FAS Research Computing

### CODE REVIEW

File: topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID: 1106
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  # Resource Name should be a short descriptor of the resource.
+  # e.g. the Center for High Throughput Computing's GlideinWMS Frontend is "CHTC-glidein2"
+  # Resource Names need to be unique across all resources in the OSG.
+  <RESOURCE NAME>:
```
Comment (edquist) [inline topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml:19]: Need to pick a resource name here.

File: topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+      # Security Contact is one to three people to contact regarding security issues
+      Security Contact:
+        Primary:
+          Name: <FIRSTNAME> <LASTNAME>
+          ID: <EMAIL HASH>
```
Comment (edquist) [inline topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml:49]: Will this resource have a security contact?

**Review by edquist — CHANGES_REQUESTED** [review]
I added missing IDs and fixed up the yaml formatting for the Squid service section.  There are just a couple items that still need attention: resource name and the service contact section.

File: topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml (hunk 1 above)
Comment (dancaunt) [reply topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml:19]:
```suggestion
  <RESOURCE NAME>: Harvard FAS Research Computing frontier-squid
```

File: topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml (hunk 2 above)
Comment (dancaunt) [reply topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml:49]:
```suggestion
      Security Contact: Harvard FAS Research Computing Security Team
        Primary:
          Name: Warren Frame
          ID: <EMAIL HASH>
```

**Review by edquist — APPROVED** [review]

**Pull request merged**